core/src/test/java/org/mule/el/mvel/MVELMapHandlingTestCase.java

```diff
@@ -24,12 +24,18 @@
     private static final String VALUE = "MG";
     private ExpressionLanguage el;
 
+    @Override
+    protected void doSetUpBeforeMuleContextCreation() throws Exception
+    {
+        setStartContext(true);
```

Comment:
so that a registry is added. This will probably no longer be required if we get rid of the limbo as we just discussed